src/test/java/teammates/test/cases/action/AdminEmailComposePageActionTest.java

```diff
@@ -91,7 +93,10 @@ public void testExecuteAndPostProcess() throws Exception {
     }
 
     @Override
+    @Test
     protected void testAccessControl() throws Exception {
-        //TODO: implement this
+        dataBundle = getTypicalDataBundle();
```

Comment:
Do update all the places where you can now just use `typicalBundle`, instead of getting it again.